.sisyphus/plans/dp-07-doc-consolidation.md

```
@@ -0,0 +1,230 @@
+   e. Move `/availability/{id}/nodes` route; add feature tests.
+   f. Add `// TODO dp-13:` annotation to `tests/Feature/SetupWizardValidationTest.php:17-21`.
+   g. Run `../../../vendor/bin/phpunit --configuration phpunit.xml` from the extension dir.
+8. Commit Phase 4.
+9. Push and open PR:
+
+```bash
+git push -u origin dp-07-doc-consolidation
+gh pr create --base dynamic-slider --title "docs+chore: consolidate docs, narrow decisions, retire dead state/alias" --fill
+```
+
+10. Run the standard CodeRabbit review loop per `/loop` semantics until merged.
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 10; Style: 1/1/1

(MD029, ol-prefix)